{
    "name": "client",
    "context": {
        "trace_id": "0x0c9a16a33b4c7c0705323b6fe9f229b8",
        "span_id": "0x471cd9ed33601944",
        "trace_state": "[]"
    },
    "kind": "SpanKind.INTERNAL",
    "parent_id": null,
    "start_time": "2023-11-19T20:58:56.795285Z",
    "end_time": "2023-11-19T20:58:56.803131Z",
    "status": {
        "status_code": "ERROR",
        "description": "FileNotFoundError: [Errno 2] No such file or directory: 'asdf'"
    },
    "attributes": {},
    "events": [
        {
            "name": "exception",
            "timestamp": "2023-11-19T20:58:56.803116Z",
            "attributes": {
                "exception.type": "FileNotFoundError",
                "exception.message": "[Errno 2] No such file or directory: 'asdf'",
                "exception.stacktrace": "Traceback (most recent call last):\n  File \"/Users/snehpatel/Documents/University/COSC3P95/venv/lib/python3.11/site-packages/opentelemetry/trace/__init__.py\", line 573, in use_span\n    yield span\n  File \"/Users/snehpatel/Documents/University/COSC3P95/venv/lib/python3.11/site-packages/opentelemetry/sdk/trace/__init__.py\", line 1055, in start_as_current_span\n    yield span_context\n  File \"/Users/snehpatel/Documents/University/COSC3P95/assign2/client.py\", line 80, in messaging_client\n    send_file(client_data)\n  File \"/Users/snehpatel/Documents/University/COSC3P95/assign2/client.py\", line 41, in send_file\n    file_size = os.path.getsize(file_path)\n                ^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"<frozen genericpath>\", line 50, in getsize\nFileNotFoundError: [Errno 2] No such file or directory: 'asdf'\n",
                "exception.escaped": "False"
            }
        }
    ],
    "links": [],
    "resource": {
        "attributes": {
            "telemetry.sdk.language": "python",
            "telemetry.sdk.name": "opentelemetry",
            "telemetry.sdk.version": "1.21.0",
            "service.name": "unknown_service"
        },
        "schema_url": ""
    }
}